.isort.cfg

```diff
@@ -0,0 +1,9 @@
+[settings]
+line_length=88
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+
```

Comment:
To what extent should we bias toward not having these files in order to make the whole thing seem simpler?

If we decide that these help make the example feel more realistic, it might be good to add headers that indicate that including them is optional.

Reply:
> To what extent should we bias toward not having these files in order to make the whole thing seem simpler?

We in part need these files for our codebase to actually work, like for `./pants lint ::` to pass in CI.

And it's rare to use isort and Flake8 without some level afaict, so I think it's useful too.

I'll add the headers, though.